Css/style.css

```diff
@@ -366,153 +460,164 @@ img {
     margin: 0 25px;
   }
 
-  /* the contact me section */
-  .contactMe {
-    background-image: url('../images/illustration-contact-form.jpg');
-    background-repeat: no-repeat;
-    width: 100%;
-    background-position: 100% 80%;
+  .sec-text {
+    margin-left: 12px;
+    margin-right: 12px;
   }
 
-  .form {
-    width: 100%;
-    max-width: 768px;
+  .sec-description {
+    padding-left: 8px;
+    padding-right: 8px;
   }
 
-  .form ul {
-    list-style: none;
-    display: flex;
-    flex-direction: column;
-    align-items: center;
-    row-gap: 24px;
+  .hero-inner {
+    margin-bottom: 0;
   }
 
-  .form ul li {
-    width: 100%;
+  .hero-title {
+    padding-top: 127px;
   }
+}
 
-  input,
-  textarea {
-    background: #fff;
-    border: 1px solid #d0d9d4;
-    border-radius: 4px;
-    width: 100%;
-    padding: 15px 16px;
-    font-size: 15px;
-    line-height: 24px;
-    color: #6b778c;
-    font-family: 'Inter', sans-serif;
+@media (max-width: 420px) {
+  .skill-item,
+  .works-item {
+    width: 90%;
+    margin: 0 auto;
+  }
+}
+
+@media (max-width: 360px) {
+  .skill-item,
+  .works-item {
+    width: 80%;
+    margin: 0 auto;
   }
+}
 
-  textarea {
-    border-color: #60c095;
+@media (max-width: 324px) {
+  .skill-item,
+  .works-item {
+    width: 70%;
+    margin: 0 auto;
   }
+}
 
-  input:focus-visible,
-  textarea:focus-visible {
-    outline-color: #27c080;
+@media (min-width: 768px) {
```

Comment:
- [ ] As the previous reviewer pointed out, you have added too many breakpoints which weren't required by the project. It is not a bad feature, but the project guidelines explicitly require the desktop breakpoint to be in the range of `992px` - `1100px`, please adjust your desktop breakpoint to have a correct value.

Reply:
![image](https://user-images.githubusercontent.com/81693905/154735691-2b6969b0-e255-4af8-ab30-d87bed21ee18.png)